doc/Roadmap.md

```diff
@@ -46,22 +48,26 @@ Please contact us and tell us about what features and improvements do you see as
 * Basic one-to-one VoIP (NAT traversal and media relaying with ICE/STUN/TURN server)
 * Contributions to ecosystem
 
-## MongooseIM 3.0.x: real-time planetary scale (2018Q1)
+### MongooseIM 3.x: planetary scale
```

Comment:
I really think we should mention geoclustering. Planetary scale is not clear enough